frontend/src/components/dialogs/SecretDialogOpenstack.vue

```diff
@@ -138,16 +162,19 @@ SPDX-License-Identifier: Apache-2.0
 <script>
 import { mapGetters } from 'vuex'
 import SecretDialog from '@/components/dialogs/SecretDialog'
-import { required, requiredIf, requiredUnless, and } from 'vuelidate/lib/validators'
-import { nilIf } from '@/utils/validators'
+import { required, requiredIf } from 'vuelidate/lib/validators'
 import { getValidationErrors, setDelayedInputFocus } from '@/utils'
 import HintColorizer from '@/components/HintColorizer'
 import ExternalLink from '@/components/ExternalLink'
 
 const requiredMessage = 'You can\'t leave this empty.'
-const requiredUserPassMessage = 'You can\'t leave this empty unless application credentials are used.'
-const requiredAppCredMessage = 'Required for application credentials.'
-const authOrAppCredentialsMessage = 'Application credentials are used only if no authentication credentials are given.'
+const requiredUserPassMessage = 'Required for technical user authentication'
+const requiredAppCredentialMessage = 'Required for application credentials authentication'
+
+const AuthenticationMethod = {
+  USER: 'user',
+  APP_CREDENTIAL: 'appcredentials'
```

Comment:
The name of the constant is singular while the value is plural. I still don't get it when it should be singular and when plural. I think it should always be plural.